tools/single_use/Normalise_Consent_Data.php

```diff
@@ -307,8 +307,7 @@
     $columnName = $column['Column_name'];
```

Comment:
i am wondering if we should rename this to `NormaliZe_Consent_Data.php` to keep US/UK spelling preference consistent.

Reply:
sure